src/components/Input/Toggle/Toggle.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+import TOGGLE_SIZES from "./constants";
+import {
+  ActiveArea,
+  ReactToggle,
+  ReactToggleTrack,
+  ReactToggleThumb,
+  ReactThumbCenteringContainer
+} from "./Toggle.styles";
+
+export default class Toggle extends Component {
+  handleClick = e => {
+    e.preventDefault();
+    const { onClick, onToggle, value, disabled } = this.props;
+
+    if (disabled) {
+      return;
+    }
+    onClick(e);
+    onToggle(value);
+    this.input.focus();
+  };
+
+  handleKeyPress = e => {
+    const { onToggle, value, onKeyPress } = this.props;
+    if (onKeyPress) {
+      onKeyPress(e);
+      this.input.focus();
+    } else {
+      const SPACE_CHAR_CODE = 32;
```

Comment:
Since we're using this constant here, in `/src/components/Input/DropDown/DropDownGroup`, `/src/components/Input/DropDown/DropDownOption`, and `/src/components/KeyboardNavigation/Provider`, it would probably make sense to define it in a single location

Reply:
Agree. I defined key char codes in a single file.